yml_to_requirements_convert.py

```diff
@@ -0,0 +1,25 @@
+import yaml
+
+
+def yml_convertor():
+
+    yml_env = yaml.safe_load(open("./environment.yml"))
+
+    requirements = []
+    for dep in yml_env["dependencies"]:
+        if isinstance(dep, str):
+            if "=" in dep:
+                package, package_version = dep.split("=")
+                if package == "python":
+                    requirements.append(package + "==3.6.2")
+                else:
+                    requirements.append(package + "==" + package_version)
+            else:
+                requirements.append(dep)
+        elif isinstance(dep, dict):
+            for preq in dep.get("pip", []):
+                requirements.append(preq)
+
+    with open("./requirements.txt", "w") as fp:
+        for requirement in requirements:
+            print(requirement, file=fp)
```

Comment:
And this is not needed.

Reply:
Anything containing environment.yml is no longer needed no.